Cargo.toml

```diff
@@ -12,22 +12,22 @@ edition = "2018"
 
 [dependencies]
 atty = "0.2"
-color-eyre = "0.6.1"
-eyre = "0.6.7"
+color-eyre = "0.6"
+eyre = "0.6"
 home = "0.5"
-lazy_static = "1.0"
-libc = "0.2.104"
+lazy_static = "1"
 rustc_version = "0.4"
 toml = "0.5"
 which = { version = "4", default_features = false }
-shell-escape = "0.1.4"
-serde_json = "1.0.48"
+shell-escape = "0.1"
+serde_json = "1"
 
 [target.'cfg(not(windows))'.dependencies]
 nix = "0.23"
+libc = "0.2.104"
```

Comment:
Why not using `0.2` here as well?

Reply:
I think it's worth using it here, since libc has been `0.2` for a while. I don't have a strong preference though (except for what I previously stated ;) )

Reply:
I think it is not a matter of taste but constancy. Or we use it for all or for none :-)

Reply:
alright fine, I'll cave in 😁